microsoftgraph/microsoft-graph-explorer-v2
# Decodes data encoded as a base64 token

## Overview

Passes transient state to Graph Explorer as a base64 encoded token.

### Demo

N/A

### Notes

N/A

## Testing Instructions

* Launch Graph Explorer
* Copy paste the url below and reload
```
http://localhost:3000/?query=eyJzYW1wbGVVcmwiOiJodHRwczovL2dyYXBoLm1pY3Jvc29mdC5jb20vdjEuMC9tZS9tZXNzYWdlcyIsInNhbXBsZVZlcmIiOiJwb3N0Iiwic2FtcGxlQm9keSI6eyJuYW1lIjoiU29tZW9uZSBJbXBvcnRhbnQifSwic2FtcGxlSGVhZGVycyI6eyJjb250ZW50LXR5cGUiOiJhcHBsaWNhdGlvbi9qc29uIn19

The base64 token above is generated from 
{
     'sampleUrl': 'https://graph.microsoft.com/v1.0/me/messages',
     'sampleVerb': 'post',
     'sampleBody': {
         'name': 'Someone Important',
     },
     'sampleHeaders': {
         'content-type': 'application/json'
     }
 }
```
* Observe that the sample is `https://graph.microsoft.com/v1.0/me/messages`
